tests/test_quotes.py

```diff
@@ -0,0 +1,33 @@
+from datetime import date
+
+from app.database.models import Quote
+from app.internal.quotes import daily_quotes
+
+DATE = date(2021, 1, 1)
+DATE2 = date(2021, 1, 2)
+
+
+def insert_quotes(session):
+    quote = Quote(
+        id=1, text='You have to believe in yourself.', author='Sun Tzu')
+    quote2 = Quote(id=2, text='Wisdom begins in wonder.', author='Socrates')
+    session.add(quote)
+    session.add(quote2)
+    session.commit()
```

Comment:
When you delete them from db?

Reply:
It's will be deleted automatically, after the session is over. (According to the fixture- 'session' in conftest).

Reply:
I'm pretty sure you need to add something like `session.delete(quote)` in order to delete it.
Fixtures are magical, but there is a limit to magic :)